js/McqView.js

```diff
@@ -19,17 +19,13 @@ class McqView extends QuestionView {
   }
 
   onKeyPress(event) {
-    if (event.which !== 13) return;
-    // <ENTER> keypress
+    if (![13, 32].includes(event.which)) return;
+    // <ENTER> or <SPACE> keypress
```

Comment:
For code clarity, use [event.key](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key).

```suggestion
    if (!['Enter', ' '].includes(event.key)) return;
```